### DISCUSSION TITLE
How to generate custom text like these -> █▀▄▀█ █▀█ █▄░█ █ ▀█▀ █▀█ █▀█ █░▀░█ █▄█ █░▀█ █ ░█░ █▄█ █▀▄    ?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [prasanthrangan/hyprdots#346] [Q&A] [answered]
What is the program name ? font name?

**Comment by KAGEYAM4** [comment]
█▀▄▀█ █▀█ █▄░█ █ ▀█▀ █▀█ █▀█
█░▀░█ █▄█ █░▀█ █ ░█░ █▄█ █▀▄

**Reply by zer0cha0s** [reply, marked as answer]
Link provided by the dev for the text generator in this issue - https://github.com/prasanthrangan/hyprdots/issues/269
